x/ibc/04-channel/cli.go

```diff
@@ -0,0 +1,81 @@
+package ics04
```

Comment:
Why is this whole file commented out?

Reply:
is the legacy code that uses the store accessors. I'll remove it